CHANGELOG.md

```diff
@@ -8,7 +8,7 @@ The crates in this repository do not adhere to [Semantic Versioning](https://sem
 
 ## Unreleased
 
-## [2.1.0]
+## [3.0.0]
```

Comment:
Just to make sure I understand, we were going to release a 2.1, but since we bumped the SGX SDK/enclaves we're instead doing a major version

Reply:
Yep, exactly.